configs/mmfashion/mask_rcnn_r50_fpn_1x.py

```diff
@@ -0,0 +1,15 @@
+_base_ = [
```

Comment:
Follow the [naming conventions](https://mmdetection.readthedocs.io/en/latest/config.html#config-name-style) to name the config file.

Reply:
Renamed the config file following the naming conventions in the new commit.